mongomock/collection.py

```diff
@@ -354,6 +354,51 @@ def _copy_field(self, obj, container):
         else:
             return copy.copy(obj)
 
+    def _extract_projection_operators(self, fields):
+        """Removes and returns fields with projection operators."""
+        result = {}
+        allowed_projection_operators = {'$elemMatch'}
```

Comment:
I don't know who still uses Python 2.6, but the set syntax with curly braces is not supported there. You'll have to write `set(['$elemMatch'])`.